README.md

```diff
@@ -10,48 +10,128 @@ hierarchies and multiple types of execution resources. It currently can use
+- [Programming guide](https://kokkos.org/kokkos-core-wiki/programmingguide.html): contains in "narrative" form a technical description of the programming model, machine model, and the main building blocks like the Views and parallel dispatch.
+
+- [API reference](https://kokkos.org/kokkos-core-wiki/): organized by category, i.e., [core](https://kokkos.org/kokkos-core-wiki/API/core-index.html), [algorithms](https://kokkos.org/kokkos-core-wiki/API/algorithms-index.html) and [containers](https://kokkos.org/kokkos-core-wiki/API/containers-index.html) or, if you prefer, in [alphabetical order](https://kokkos.org/kokkos-core-wiki/API/alphabetical.html).
+
+- [Use cases and Examples](https://kokkos.org/kokkos-core-wiki/usecases.html): a series of examples ranging from how to use Kokkos with MPI to Fortran interoperability.
+
+## Quick Start for the impatient
+
+### Obtaining Kokkos
+
+The latest release of Kokkos can be obtained from the [GitHub releases page](https://github.com/kokkos/kokkos/releases/latest)
+or programmatically using curl to get the latest release tarball and extract it into a directory called `kokkos`:
+
+```bash
+mkdir kokkos && curl -s -L https://github.com/kokkos/kokkos/archive/refs/tags/4.2.00.tar.gz | tar xz - -C kokkos --strip-components 1
+```
+
+To clone the latest development version of Kokkos from GitHub:
+
+```bash
+git clone -b develop  https://github.com/kokkos/kokkos.git
+```
+
+### Building Kokkos
+
+To build Kokkos, you will need to have a C++ compiler that supports C++14 or later.
+All requirements including minimum and primary tested compiler versions can be found [here](https://kokkos.org/kokkos-core-wiki/requirements.html).
+
+Building and installation instructions are described [here](https://kokkos.org/kokkos-core-wiki/building.html).
+
+#### Building for CPU on Linux or macOS
+
+To use the OpenMP backend, you will need a compiler that supports OpenMP, such as GCC, Clang, or Intel C++ Compiler.
+On MacOS, you will need to install a recent version of `libomp` for example using Homebrew: `brew install libomp`.
+
+To build Kokkos targetting OpenMP for CPU on Linux or macOS, you can use the following commands:
+
+```bash
+cd kokkos
+cmake -B build -S . -DKokkos_ENABLE_OPENMP=On -DCMAKE_INSTALL_PREFIX=/opt/kokkos
+cmake --build build && cmake --install build
+```
+
+Note that you can also use `Kokkos_ENABLE_THREADS` to enable the use of threads instead of OpenMP.
+
+#### Building for CUDA on Linux
+
+To build Kokkos targetting CUDA on Linux, you can use the following commands:
+
+```bash
+cd kokkos
+cmake -B build -S . -DKokkos_ENABLE_CUDA=On -DCMAKE_INSTALL_PREFIX=/opt/kokkos
+cmake --build build && cmake --install build
+```
+
+#### Building for HIP on Linux
+
+To build Kokkos targetting HIP on Linux, you can use the following commands:
+
+```bash
+cd kokkos
+cmake -B build -S . -DKokkos_ENABLE_HIP=On -DCMAKE_INSTALL_PREFIX=/opt/kokkos
+cmake --build build && cmake --install build
+```
```

Comment:
I would push all of this to https://kokkos.org/kokkos-core-wiki/building.html and provide instructions for all backends there instead.

Reply:
The idea was to have an online QuickStart guide directly with Kokkos, which can be copied and pasted. A broader documentation effort is in kokkos/kokkos-core-wiki#500 .